apps/workspaces/views.py

```
@@ -66,6 +65,37 @@ def get(self, request):
+
+        if not netsuite_credentials:
+         status_code = status.HTTP_400_BAD_REQUEST
+         message = "Netsuite credentials not found"
+        elif netsuite_credentials.is_expired:
+         status_code = status.HTTP_400_BAD_REQUEST
+         message = "Netsuite connection expired"
+
+        try:
+         netsuite_connection = NetSuiteConnector(netsuite_credentials=netsuite_credentials, workspace_id=workspace_id)
+         netsuite_connection.connection.locations.count()
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)